examples/stable/kafka/s3ConnectorConfiguration.properties

```diff
@@ -0,0 +1,13 @@
+partitioner.class= io.confluent.connect.storage.partitioner.DefaultPartitioner
+connector.class= io.confluent.connect.s3.S3SinkConnector
+tasks.max= 1
+# topic Name
+topics= blogpost
+s3.part.size= 5242880
+# flush size
+flush.size= 3
+storage.class= io.confluent.connect.s3.storage.S3Storage
+format.class= io.confluent.connect.s3.format.json.JsonFormat
+schema.compatibility= NONE
+# connector name
+name= kafka-s3
```

Comment:
and new line at the end of this file and other files, where we see this mark.

Reply:
added a new line